// Copyright(c) 2019 Zededa, Inc.
// All rights reserved.

syntax = "proto3";
package eve_tools;

message File {
    string Name = 1;
    bytes  Content = 2;
}

message EveTPMRequest {
    string Command = 1;
    repeated string ExpectedFiles = 2;
    repeated File InputFiles = 3;
}

message EveTPMResponse {
    bytes Response = 1;
    repeated File OutputFiles = 2;
}
